### PR TITLE
notification sound effect

### PR DESCRIPTION
**Pull request by lepras**
in reference to [this](https://log.asteroidos.org/?date=2022-11-25&level=10#104) chat on matrix.

### CODE REVIEW

File: qml/quicksettings/QuickSettings.qml [hunk 1]
```diff
@@ -63,13 +66,13 @@ Item {
     }
 
     QuickSettingsToggle {
-        id: lockedToggle
+        id: soundToggle
```
Comment (MagneFire) [inline qml/quicksettings/QuickSettings.qml:69]: These changes should not be part of this PR.

We should discuss in a separate issue. Obviously it would be nice to have a quick button to mute all sounds, but not every watch has a speaker rendering this button entirely useless.
It also replaces an already existing function, which might not be desirable right now.

**Review by MagneFire — CHANGES_REQUESTED** [review]
Thank you for this PR! And I'm sorry for taking this long to review it.

Please see the feedback on this PR.

When you remove the QuickSettings changes I think this PR should be ready to merge.
However, I think it would be better to eventually add these sound related events to the `ngfd` events: https://github.com/AsteroidOS/meta-asteroid/blob/master/recipes-nemomobile/ngfd/ngfd/events.d/alarm.ini#L1 and add some of the lines references here: https://github.com/sailfishos/ngfd/blob/master/data/events.d/clock.ini#L2.
We could even take advantage of profiles to alter the behaviour of alarm events: https://github.com/sailfishos/ngfd/blob/master/data/events.d/ringtone.ini#L4 this would remove the need to dynamically change the system volume.

File: qml/quicksettings/QuickSettings.qml (hunk 1 above)
Comment (lepras) [reply qml/quicksettings/QuickSettings.qml]: removed that commit

**Comment by lepras** [comment]
re-synced to the upstream for merging.

**Review by MagneFire — APPROVED** [review]
